lib/spritesheet.dart

```diff
@@ -36,12 +37,11 @@ class SpriteSheet {
     int x,
     int y,
   ) {
+    final size = Vector2(textureWidth.toDouble(), textureHeight.toDouble());
```

Comment:
not sure if we can have extension constructors, but would be nice to have a Vector2.fromInts like we had for Position

Reply:
We can not, unfortunately.

Reply:
😢 

we can do a static function then like Vector2Extension.fromInts. but let's think that through for a followup